Doc/library/typing.rst

```diff
@@ -1161,7 +1161,8 @@ These are not used in annotations. They are building blocks for creating generic
     Usage::
 
       T = TypeVar('T')  # Can be anything
-      A = TypeVar('A', str, bytes)  # Must be str or bytes
+      S = TypeVar('S', bound=str)  # Can be any subtype or subclass of str
```

Comment:
```suggestion
      S = TypeVar('S', bound=str)  # Can be any subtype of str
```
I don't know that we're doing a great job with this terminology, but I think every subclass is also a subtype. Mentioning both invites confusion.

Reply:
Yeah, the reason I went with both is that "subtype" is more accurate than subclass (e.g. the proposed `LiteralString` type would be a subtype but not a subclass), but "subtype" is a term that (I think) is foreign to most Pythonista, especially those new to typing.

Not really sure what the answer is, but I think you're right that we should just go with "subtype" here, in any event 👍